sqswatcher/sqswatcher.py

```diff
@@ -308,8 +289,9 @@ def _poll_queue(scheduler, region, cluster_user, queue, table, proxy_config, max
                         log.info("Unable to query scheduler configuration, discarding %s message" % event_type)
 
                     message.delete()
-
-            results = queue.receive_messages(MaxNumberOfMessages=10)
```

Comment:
`while len(results) > 0:` will continue to work with the same `results`.

Reply:
Good catch! this part of the logic will change completely as part of the upcoming PRs so I'll just leave it as is.